proto/poktroll/migration/query.proto

```diff
@@ -1,31 +1,45 @@
 syntax = "proto3";
+
 package poktroll.migration;
 
 import "amino/amino.proto";
 import "gogoproto/gogo.proto";
 import "google/api/annotations.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
 import "poktroll/migration/params.proto";
+import "poktroll/migration/types.proto";
 
 option go_package = "github.com/pokt-network/poktroll/x/migration/types";
 option (gogoproto.stable_marshaler_all) = true;
 
 // Query defines the gRPC querier service.
 service Query {
+  
   // Parameters queries the parameters of the module.
-  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+  rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/pokt-network/poktroll/migration/params";
+  
+  }
+  
+  // Queries a MorseAccountState by index.
+  rpc MorseAccountState (QueryGetMorseAccountStateRequest) returns (QueryGetMorseAccountStateResponse) {
+    option (google.api.http).get = "/pokt-network/poktroll/migration/morse_account_state";
+  
   }
 }
-
 // QueryParamsRequest is request type for the Query/Params RPC method.
 message QueryParamsRequest {}
 
 // QueryParamsResponse is response type for the Query/Params RPC method.
 message QueryParamsResponse {
+  
   // params holds all the parameters of this module.
-  Params params = 1 [
-    (gogoproto.nullable) = false,
-    (amino.dont_omitempty) = true
-  ];
-}
\ No newline at end of file
+  Params params = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+}
+
+message QueryGetMorseAccountStateRequest {}
+
+message QueryGetMorseAccountStateResponse {
```

Comment:
E2E test for this? Doesn't have to be in this PR.

Reply:
We have unit/integration coverage over this.